packages/plugins/opentelemetry/package.json

```diff
@@ -52,6 +52,7 @@
     "graphql": "^15.9.0 || ^16.9.0"
   },
   "dependencies": {
+    "@graphql-hive/core": "0.13.0-alpha-20250707104821-802de2e836e8fc0fb59533a3c4c63cf53e9fc33c",
```

Comment:
Do we want the alpha release here?

Reply:
No, we will have to wait for this PR to be merged: https://github.com/graphql-hive/console/pull/6876

Reply:
`@graphql-hive/core@0.13.0` got released: https://github.com/graphql-hive/console/pull/6869